src/main/java/io/vertx/core/http/impl/Http2ServerRequest.java

```diff
@@ -299,6 +299,13 @@ public HttpServerRequest pause() {
     return this;
   }
 
+  @Override
+  public boolean isPaused() {
+    synchronized(conn) {
+      return super.isPaused();
```

Comment:
This only takes the `VertxHttp2Stream.isPaused` and wraps it in a `synchronized(conn)` block. Should perhaps have different names for this and the superclass method, but couldn't come up with a good parallel to `doPause` - `doIsPaused`?